spec/models/msg_target_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe MsgTarget, type: :model do
```

Comment:
If we are not testing it, it's better to remove the file.

Reply:
Let's clean up the tests in the next feature branch, okay?